components/List/index.js

```diff
@@ -2,26 +2,27 @@ import styled from "styled-components";
 import Link from "next/link";
 import Image from "next/image";
 
-export default function List({players}) {
-
+export default function List({ players }) {
   return (
     <StyledList>
       {players.map((player) => {
         return (
-          <Link
-            href={`/players/${player._id}`}
-            key={player._id}
-          >
+          <Link href={`/players/${player._id}`} key={player._id}>
             <StyledListItem>
-              <Image
-                src={player.response[0].statistics[0].team.logo}
-                height={50}
-                width={50}
-                alt={player.response[0].player.name}
-              />
-              {player.response[0].player.firstname}
-              <br />
-              {player.response[0].player.lastname}
+              <ImageContainer>
+                <Image
+                  src={player.response[0].statistics[0].team.logo}
+                  height={50}
+                  width={50}
+                  alt={player.response[0].player.name}
+                />
+              </ImageContainer>
+              <TextContainer>
+                {player.response[0].player.name}
```

Comment:
Do you only want to display the first result? Or would it make sense to map over all of them?

Reply:
This is OK, I'm already mapping over all players in line 8 of my code.